aeron-cluster/src/main/java/io/aeron/cluster/EgressPublisher.java

```diff
@@ -128,4 +130,41 @@ boolean newLeader(
 
         return false;
     }
+
+    boolean sendAdminResponse(
+        final ClusterSession session,
+        final long correlationId,
+        final AdminRequestType adminRequestType,
+        final AdminResponseCode responseCode,
+        final String message)
+    {
+        final int length = MessageHeaderEncoder.ENCODED_LENGTH +
+            AdminResponseEncoder.BLOCK_LENGTH +
+            AdminResponseEncoder.messageHeaderLength() +
+            message.length() +
+            AdminResponseEncoder.payloadHeaderLength();
+
+        int attempts = SEND_ATTEMPTS;
+        do
+        {
+            final long result = session.tryClaim(length, bufferClaim);
+            if (result > 0)
+            {
+                adminResponseEncoder
+                    .wrapAndApplyHeader(bufferClaim.buffer(), bufferClaim.offset(), messageHeaderEncoder)
+                    .clusterSessionId(session.id())
+                    .correlationId(correlationId)
+                    .requestType(adminRequestType)
+                    .responseCode(responseCode)
+                    .message(message)
```

Comment:
Try claim should not be used when the message count be potentially fragmented due to variable length data.